server/routes/overview/overviewController.ts

```diff
@@ -154,23 +136,4 @@ export default class OverviewController {
     const view = new TimelineView(prisonerSummary, timeline)
     res.render('pages/overview/index', { ...view.renderArgs })
   }
-
-  async setPrisonNamesOnEducationRecords(
```

Comment:
The prison name lookup is done by the service layer, our controller doesnt need to worry about it 👍